Resources/doc/reference/form_field_definition.rst

```diff
@@ -227,16 +227,23 @@ You can easily add a new ``galleryHasMedias`` row by defining one of these optio
                 ->add('enabled')
                 ->add('name')
                 ->add('defaultFormat')
-                ->add('galleryHasMedias', 'sonata_type_collection', array(), array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                    'sortable' => 'position',
-                    'limit' => 3
+                ->add('galleryHasMedias', 'sonata_type_collection', array(
```

Comment:
I'm looking for anything that could show me what has changed between sf2 and 3 in the form element names... this dosn't appear clearly in the changelog.
Sorry I don't know that much sf to understand what @core23 means by symfony 3 style.